_posts/en/2019-12-20-why-we-are-promoting-risks.md

```diff
@@ -0,0 +1,95 @@
+
+The [Financial Administration Act](https://laws.justice.gc.ca/eng/acts/F-11/index.html) contains the following 2 sections:
+
+section **7 (1) (c)**
+
+> 7 (1) The Treasury Board may act for the Queen’s Privy Council for Canada on all matters relating to
+>
+> (c) financial management, including estimates, expenditures, financial commitments, accounts, fees or charges for the provision of services or the use of facilities, rentals, licences, leases, revenues from the disposition of property, and procedures by which departments manage, record and account for revenues received or receivable from any source whatever;
+
+section **16.4 (1)**
+
+> 16.4 (1) Within the framework of the appropriate minister’s responsibilities and his or her accountability to Parliament, and subject to the appropriate minister’s management and direction of his or her department, the accounting officer of a department named in Part I of Schedule VI is accountable before the appropriate committees of the Senate and the House of Commons for
+>
+> (a) the measures taken to organize the resources of the department to deliver departmental programs in compliance with government policies and procedures;
+>
+> (b) the measures taken to maintain effective systems of internal control in the department;
+
+The above three points in this blog (promoting risks, incorrect investment decisions, and diluting empathy towards the end-user) are arguments that our fiduciary responsibility towards public funds needs to improve. We must find methods that provides a much simpler, faster, and agile way of releasing funds for digital projects. [Other countries, like Australia](https://www.itnews.com.au/news/federal-it-funding-shake-up-floated-by-services-australia-minister-534843) are looking at new funding models that provides the means to support Digital Transformation, such as
+
+> enable agencies to try things, learn and scale up or share their learnings before significant amounts of money and reputation capital are sunk into projects that may not deliver what they set out to do.
+
+## Recommendations
+
+To remediate the above problems, I propose the following:
+
+1. Promote smaller IT investments over large ones as a means to reduce risk impact to the organization;
+2. Treat every project as a hypothesis, one that must be validated by end-users as fast as possible, which will invariably require course corrections;
+3. Put much more emphasis on outcome measurements than on project deliverables. This promotes investment in [Business Architecture and Departmental Value Stream](http://dialogue/grp/EA-AE/SitePages/dvsmodel.aspx) using strong, reliable metrics from lines of businesses;
+4. Treat software like products, not physical assets. They need a steady source of funding and dedicated teams to allow timely technical debt remediation and evolution. Teams should be comprised not only of developers but also security, operations, testing, policy experts, and *actual* users;
```

Comment:
4. Treat software like products, not physical assets. They need a steady source of funding and dedicated teams to allow timely technical debt remediation and evolution. Teams should be comprised not only of developers but also security, operations, testing, policy experts, and *actual* users;

Actual users? On the development team? If the clients are internal (being paid by the same organization as the development team), then sure. If the clients are citizens, for example, we should most definitely consult them regularly, though I wouldn't say "on the development team", in this case.

Reply:
I see what you mean there. I think the intended point here was to develop with actual users, not "with users in mind". Might need a slightly different syntax.

Reply:
I say actual users. This to further promote that IT is no longer a backoffice function and that we're moving away from a contracting-model relationship and towards a partnership one.
I'm also not saying on the development team. I'm saying team. Dev is but one function of that team.

Room to interpret actual users though. Product Owner maybe. But as long as "the business" represents a group that actually uses the software. Not an intermediary.

Reply:
will rephrased to say "that works with actual users"